### PR TITLE
Bug 1199622 Update PDF links for content services

### PR DESCRIPTION
**Pull request by sgarrity**

### CODE REVIEW

File: bedrock/mozorg/templates/mozorg/contentservices/start.html [hunk 1]
```diff
@@ -25,8 +25,8 @@ <h1 class="title-shadow-box">{{ self.page_title() }}</h1>
     Please review our <a href="{{ url }}">Frequently Asked Questions</a> page.
   {% endtrans %}
   {{ _('We also have a set of creative guidelines, which cover everything you need to know to build your own Tile.') }}
-  {% trans url='http://static.mozilla.com/moco/en-US/pdf/Partner_Specs_Directory_Tiles.pdf', url2='http://static.mozilla.com/moco/en-US/pdf/Partner_Specs_Suggested_Tiles.pdf' %}
-    Download for <a href="{{ url }}">Directory Tiles</a> or <a href="{{ url2 }}">Suggested Tiles</a>.
+  {% trans url='http://static.mozilla.com/moco/en-US/pdf/Creative_Specs_for_Directory_Suggested_Tiles.pdf' %}
+    Download our creative guidelines for<a href="{{ url }}">Directory Tiles and Suggested Tiles</a>.
```
Comment (schalkneethling) [inline bedrock/mozorg/templates/mozorg/contentservices/start.html]: Missing space after `for`

Comment (sgarrity) [inline bedrock/mozorg/templates/mozorg/contentservices/start.html]: Fixed, thanks.

**Comment by sgarrity** [comment]
@schalkneethling I don't think these pages are translated.

**Comment by schalkneethling** [comment]
r+ :cake:

**Pull request merged**